tests/e2e/helpers/api/api-database.ts

```diff
@@ -0,0 +1,73 @@
+
+import { t } from 'testcafe';
+import * as request from 'supertest';
+import { asyncFilter, doAsyncStuff } from '../async-helper'
+import { AddNewDatabaseParameters, OSSClusterParameters } from '../../pageObjects/add-redis-database-page';
+import { apiUrl } from '../../helpers/conf';
+
+const endpoint = apiUrl;
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'; // lgtm[js/disabling-certificate-validation]
```

Comment:
it is an environment variable for avoiding an error  'unable to verify the first certificate'. This works to ignore the certificate verification when testing HTTPS against localhost. 
I've took this way from integration tests
RedisInsight\redisinsight\api\test\helpers\server.ts

Reply:
I see it throws an alert. Could you think of an alternative solution?

Reply:
Can't you call it via the CI system instead of within the test?

Reply:
If you mean to use CircleCI env variables for it - we will need to do the same on local machines for all devs then which is incorrect. Also, it is incorrect to set it on CircleCI env variables because this rule will be applied to all other parts of application
If you suggest to invoke tests with inline env variable like `ENV=val ./something` in circleci.yml or even using .env file we have to store it in the source code anyway and code analysis tool will still find this.

Reply:
@ofersteinberg It has. This is what I was talking about. 
In this case we will remove this row in the code and move it to the vault. As a result in my local PC tests will fail. I have to add such env to my local machine too then (each dev and qa have to do it). But I don't want to care about it. I want all tests are running out-of-the box without additional steps to do.